CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,3 @@
+- Fix: Added try-catch error handling to functions like jsonparse, jsonstringify, toisodate, etc. (#1702).
+- Fix: Modified functions (toisostring, gettime, parseint, etc.) to return null instead of NaN (#1702).
+- Add: Added tests for error scenarios, including a specific test for expressionParser.parse() (#1702).
```

Comment:
The changelog should be redacted in a user-perspective. In this sense I'd suggest to simplify to:

```suggestion
- Fix: modified JEXL transformations (toisostring, gettime, parseint, etc.) to return null instead of NaN when some unexpected situation occurs (#1702).
```

Reply:
Note that, if possible, the reference should be to the issue associated to the PR, not to the PR itself.